src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
```

Comment:
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {
```

create interface Drawable with method `void draw();`